tools/roxvet/common/file_filters.go

```diff
@@ -15,14 +15,15 @@ func IsGeneratedFile(f *ast.File) bool {
 	if len(f.Comments) == 0 {
```

Comment:
don't need the short-circuit here since we use `for range` below to iterate over the list of comments.